src/main/pages/che-7/installation-guide/proc_upgrading-che-using-operatorhub.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// upgrading-{prod-id-short}
+
+[id="upgrading-{prod-id-short}-using-operatorhub_{context}"]
+= Upgrading {prod-short} using OperatorHub
+
+This section describes how to upgrade from previous minor version using the Operator from OperatorHub in the OpenShift web console.
+
+.Prerequisites
+
+* An administrator account on an OpenShift instance.
+
+* An instance of a previous minor version of {prod-short}, installed using an Operator from OperatorHub on the same instance of OpenShift.
```

Comment:
I think it can be simplified without losing too much meaning

>{prod-short} is installed using an Operator on your OpenShift cluster/instance.
or if we introduce attribute
>{prod-short} {prev-minor-ver} is installed using an Operator on your OpenShift cluster/instance.

Reply:
The whole idea is to make the exact previous version disappear. Too much a pain to maintain it moves too fast.

Reply:
Ok, then I think we can get away with just 

>{prod-short} is installed using an Operator on your OpenShift instance.

We already mention the previous minor version in the title, I think it will trim the sentence a little bit.